eng/pipelines/prepare-pipelines.yml

```diff
@@ -1,5 +1,8 @@
 trigger: none
 
+variables:
```

Comment:
We don't need to do it as part of this change but I have been thinking about having a globals.yml variable template in eng/common where we could import things like this and the CG properties in all our pipelines.

Reply:
Seems reasonable.